### PR TITLE
add warning information in v1.0.3 to v1.1.0

### PR DESCRIPTION
**Pull request by FrankYang0529**

### CODE REVIEW

File: docs/upgrade/v1-0-3-to-v1-1-0.md [hunk 1]
```diff
@@ -6,6 +6,32 @@ title: ""
...
+```
+cluster_networks:
+  vlan:
+    enable: true
+    description: "harvester-mgmt"
+    config:
+      defaultPhysicalNIC: harvester-mgmt
+```
+
+Related issue: [[BUG] Harvester Upgrade 1.0.3 to 1.1.0 does not handle multiple SLAVE in BOND for management interface](https://github.com/harvester/harvester/issues/3045)
```
Comment (guangbochen) [inline docs/upgrade/v1-0-3-to-v1-1-0.md:33]: can we add `:::caution` to this section, thanks.

Comment (FrankYang0529) [reply docs/upgrade/v1-0-3-to-v1-1-0.md:33]: Update to `:::warning` block. Thanks

**Review by bk201 — APPROVED** [review]
LGTM!

**Review by guangbochen — APPROVED** [review]
LGTM, thanks.

**Pull request merged**